server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -439,4 +435,20 @@ export class TransformHandler {
         const jitteredDelay = Math.floor(Math.random() * 10)
         return exponentialDelay + jitteredDelay // returns in milliseconds
     }
+
+    logSuggestionForFailureResponse(request: GetTransformRequest, job: TransformationJob, failureStates: string[]) {
+        let status = job?.status ?? PollTransformationStatus.NOT_FOUND
+        let reason = job?.reason ?? ''
+        if (failureStates.includes(status)) {
+            let suggestion = ''
+            if (reason.toLowerCase().includes('build validation failed')) {
+                suggestion =
+                    'Please close Visual Studio, delete the directories where build artifacts are generated (e.g. bin and obj), and try running the transformation again.'
```

Comment:
Do we expect this server to be only used in visual studio and no other IDE? Otherwise I would reword this to `Please close your IDE`

Reply:
Yes, it's only used in visual studio.